api/dev_data/shipping_information.json

```diff
@@ -0,0 +1,28 @@
+{
+  "shipping_information": [
+      {
+          "id": "1",
+          "needs_shipping_address": true,
+          "sharer_pays": true,
+          "ups_code_accepted": true,
+          "fedex_code_accepted": true,
+          "reimbursement_accepted": false,
+          "other_payment_methods_accepted": false,
```

Comment:
How will payment work with other shipping methods? Will the requester just have to reach out to the owner to find out what the method is? Should there be a nullable text field to store a "other" shipping method?

Reply:
My assumption is that they would put something into the `restrictions` field? However yes, the sharing of address and/or other payment details will be done via email for the MVP.

Reply:
I think this is just to allow people to get an idea of what to expect when they reach out. And "other" is planned on being mostly unused. They can put any other restrictions details in the text box in the form.